### PR TITLE
Added basic file logger

### PR DESCRIPTION
**Pull request by njbrown**

### CODE REVIEW

File: src/core/database/database.cpp [hunk 1]
```diff
@@ -12,7 +13,8 @@
 
 Database::Database()
 {
-    if (!QSqlDatabase::isDriverAvailable(Constants::DB_DRIVER)) qCritical("DB driver not present!");
+    qDebug() << QSqlDatabase::drivers();
```
Comment (iKlsR) [inline src/core/database/database.cpp]: remove

File: src/irisgl/irisgl.pri [hunk 2]
```diff
@@ -118,7 +119,8 @@ SOURCES += \
     $$PWD/src/scenegraph/scene.cpp \
     $$PWD/src/scenegraph/scenenode.cpp \
     $$PWD/src/geometry/plane.cpp \
-    $$PWD/src/geometry/frustum.cpp
+    $$PWD/src/geometry/frustum.cpp \
```
Comment (iKlsR) [inline src/irisgl/irisgl.pri]: ???

File: src/mainwindow.cpp [hunk 3]
```diff
@@ -102,6 +103,8 @@ MainWindow::MainWindow(QWidget *parent) :
     ui(new Ui::MainWindow)
 {
     ui->setupUi(this);
+
+    iris::Logger::getSingleton()->init(getAbsoluteAssetPath("log.txt"));
```
Comment (iKlsR) [inline src/mainwindow.cpp]: use more descriptive name

File: src/irisgl/irisgl.pri (hunk 2 above)
Comment (njbrown) [inline src/irisgl/irisgl.pri]: fixed the missing "\"
look at it again

**Pull request merged**